graph/api/src/main/java/org/jboss/windup/graph/model/ProjectModel.java

```diff
@@ -182,4 +197,25 @@
      */
     @GremlinGroovy("it.out('" + PROJECT_MODEL_TO_FILE + "').has('" + FileModel.IS_DIRECTORY + "', false)")
     Iterable<FileModel> getFileModelsNoDirectories();
+
+    /**
+     * Returns the project model that represents the whole application. If this projectModel is the root projectModel, it will return it.
+     * @return ProjectModel representing the whole application
+     */
+    @JavaHandler
+    public ProjectModel getParentRootProjectModel();
```

Comment:
It is also possible for the current project to be the root project. In that case, it isn't a parent at all.